app.conf

```diff
@@ -0,0 +1,26 @@
+server {
+
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	root /var/www/app/wordpress;
+
+	index index.php index.html index.htm;
+
+	server_name localhost;
```

Comment:
It's not necessary to remove this line. It's more question to check out info.

What does this line?
What happen if you remove this line?

Reply:
Well, that line is useful when dealing with DNS stuff as it helps nginx to know which server block to use depending which domain is making the request. It also works with IP addresses.

If I delete it, everything will still work because the enclosing server block is configured to be the default server.

```
server {
	listen 80 default_server;
	listen [::]:80 default_server;
         ...

 }

```

Reply:
I think its useful to have `server_name` if you have more then one site configured on single host. Otherwise I don't see any reasons to use it.